src/components/templates/AvatarGrid/AvatarGrid.tsx

```diff
@@ -1,31 +1,27 @@
 import React, { useCallback, useEffect, useState } from "react";
-import firebase from "firebase/app";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
 
-// Components
-import { RoomModal } from "./RoomModal";
-import Announcement from "./Announcement";
-import ChatDrawer from "components/organisms/ChatDrawer";
-import UserProfileModal from "components/organisms/UserProfileModal";
-import UserProfilePicture from "components/molecules/UserProfilePicture";
+import { AvatarGridRoom } from "types/rooms";
+import { User } from "types/User";
+
+import { makeUpdateUserGridLocation } from "api/profile";
 
-// Hooks
 import { useSelector } from "hooks/useSelector";
 import { useUser } from "hooks/useUser";
 import { useVenueId } from "hooks/useVenueId";
 import { usePartygoers } from "hooks/users";
 
-// Utils | Settings | Constants
 import { WithId } from "utils/id";
 import { openRoomWithCounting } from "utils/useLocationUpdateEffect";
 import { currentVenueSelector } from "utils/selectors";
 
-// Typings
-import { AvatarGridRoom } from "types/AvatarGrid";
-import { User } from "types/User";
+import { RoomModal } from "./RoomModal";
+import Announcement from "./Announcement";
+import ChatDrawer from "components/organisms/ChatDrawer";
+import UserProfileModal from "components/organisms/UserProfileModal";
+import UserProfilePicture from "components/molecules/UserProfilePicture";
 
-// Styles
 import "./AvatarGrid.scss";
```

Comment:
[OOS] I think this template should be deleted

Reply:
See https://github.com/sparkletown/sparkle/pull/1080#discussion_r564114898